src/Tablebot/Plugins/Reminder.hs

```diff
@@ -35,14 +35,10 @@ import Database.Persist qualified as P (delete)
 import Database.Persist.TH
 import Discord.Types
 import Tablebot.Plugin
-import Tablebot.Plugin.Discord
-  ( Message,
-    getMessage,
-    sendMessageVoid,
-  )
+import Tablebot.Plugin.Discord (getMessage, sendMessageVoid)
 import Tablebot.Plugin.Parser (number, quoted, space)
 import Text.Megaparsec
-import Text.RawString.QQ
+import Text.RawString.QQ qualified as RS (r)
```

Comment:
I think this was discussed and it should be unqualified now, and the `r` variable renamed